apps/web/drizzle/0000_graceful_spectrum.sql

```diff
@@ -0,0 +1,46 @@
+CREATE SCHEMA "latitude";
+--> statement-breakpoint
+CREATE TABLE IF NOT EXISTS "latitude"."commits" (
```

Comment:
use uuids instead of auto incremental ids for identification of commits and prompts this way we can safely reference them from our api without revealing information and its easier to reference them from other services like elastic search

Reply:
well maybe for prompts is not needed although for consistency i'd use uuids everywhere? opinions?